src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/commands.py

```diff
@@ -44,13 +47,19 @@
                                 data_client_path.format('KeyVaultClient.get_keys'))
 cli_keyvault_data_plane_command('keyvault key list-versions',
                                 data_client_path.format('KeyVaultClient.get_key_versions'))
+cli_keyvault_data_plane_command('keyvault key list-deleted',
```

Comment:
Alternatively, you could simply enhance the existing list command with a --deleted flag (or something similar) rather than have a completely separate command. This would be similar to list/list-all folding that many commands do